packages/web-app-files/src/components/SideBar/Shares/Links/DetailsAndEdit.vue

```diff
@@ -371,6 +372,14 @@ export default defineComponent({
       )
     },
 
+    expirationDateHint() {
+      return this.$gettextInterpolate(
```

Comment:
Please don't use `$gettextInterpolate` anymore, you can interpolate in `$gettext` directly.